core/domain/collection_services.py

```diff
@@ -199,19 +182,24 @@ def get_collection_by_id(collection_id, strict=True, version=None):
         Collection or None. The domain object representing a collection with the
         given id, or None if it does not exist.
     """
-    collection_memcache_key = _get_collection_memcache_key(
-        collection_id, version=version)
-    memcached_collection = memcache_services.get_multi(
-        [collection_memcache_key]).get(collection_memcache_key)
-
-    if memcached_collection is not None:
-        return memcached_collection
+    sub_namespace = python_utils.convert_to_bytes(version) if version else ''
+    cached_collection = caching_services.get_multi(
+        caching_services.CACHE_NAMESPACE_COLLECTION,
+        sub_namespace,
+        [collection_id]).get(
```

Comment:
Move ) to next line, to align with opening paren. Then you can put the next line at the end of this one.